lldb/source/Plugins/Platform/POSIX/PlatformPOSIX.cpp

```diff
@@ -678,8 +678,8 @@ uint32_t PlatformPOSIX::DoLoadImage(lldb_private::Process *process,
     loaded_image->Clear();
 
   std::string path;
```

Comment:
```suggestion
  std::string path = remote_file.GetPath(false);
```
Might as well fold the assignment into the declaration.